ci/docker/powerpc64-unknown-linux-gnu/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:16.04
+
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends \
+        gcc libc6-dev qemu-user ca-certificates \
+        gcc-powerpc64-linux-gnu libc6-dev-ppc64-cross \
+        qemu-system-ppc
+
+ENV CARGO_TARGET_POWERPC64_UNKNOWN_LINUX_GNU_LINKER=powerpc64-linux-gnu-gcc \
+    CC=powerpc64-linux-gnu-gcc \
```

Comment:
Could we use the suffixed version (CC_powerpc64_...) here? My understanding is that CC also overrides the host compiler and, although we aren't using it here, I think it's better to avoid surprises in the future.

Some with the other Dockerfiles.